tool_path_planner/src/surface_walk_raster_generator.cpp

```diff
@@ -496,7 +496,11 @@ int findClosestPoint(const std::vector<double>& pt, const std::vector<std::vecto
   return index;
 }
 
-bool SurfaceWalkRasterGenerator::setConfiguration(const Config& config) { config_ = config; }
+bool SurfaceWalkRasterGenerator::setConfiguration(const Config& config)
+{
+  config_ = config;
+  return true;
```

Comment:
Good catch

Reply:
newer compiler seems to flag these as errors